packages/form-participation/test-web/FormControlMixin.lit.test.ts

```diff
@@ -0,0 +1,79 @@
+import { elementUpdated, expect, fixture, fixtureCleanup } from '@open-wc/testing';
+import { sendKeys } from '@web/test-runner-commands';
+import { FormControlMixin, formValues } from '../src';
+import { html, LitElement, TemplateResult } from 'lit';
+import { customElement, query } from 'lit/decorators.js';
+import { live } from 'lit/directives/live.js';
+
+@customElement('form-control-lit-no-validators')
+class FormControlLitNoValidators extends FormControlMixin(LitElement) {
+  @query('input')
+  validationTarget: HTMLElement;
+
+  render(): TemplateResult {
+    return html`<input
+      @input="${this._onInput}"
+      .value="${live(this.value)}"
+    >`;
+  }
+
+  _onInput({ target }: Event & { target: HTMLInputElement }) {
+    this.value = target.value;
+  }
+}
+
+describe('The FormControlMixin using LitElement', () => {
+  let form: Element;
+  let el: FormControlLitNoValidators;
+
+  const getFormValues = () => {
+    return formValues(form as unknown as HTMLFormElement);
+  };
+
+  beforeEach(async () => {
+    form = await fixture(html`
+      <form>
+        <form-control-lit-no-validators
+          name="formControl"
+        ></form-control-lit-no-validators>
+      </form>
+    `);
+    // @ts-ignore
+    el = form.querySelector<FormControlLitNoValidators>('form-control-lit-no-validators');
```

Comment:
What does TS complain about here? Do we need to place the test element on the `HTMLElementTagNameMap`?

Reply:
Nothing apparently. I'll remove it.